DOCSP-33192 Compass Generative AI Querying Content (#557)
mongodb/docs-compass: source/query-with-natural-language/enable-natural-language-querying.txt

.. _enable-natural-language-querying:

================================
Enable Natural Language Querying
================================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1

You can enable Compass natural language querying by toggling the 
:guilabel:`Use Generative AI` setting. Natural language queries can 
be a helpful starting point and assist you in learning to write MongoDB 
queries.

About this Task
---------------

Alternatively, you can enable natural language querying by clicking the 
:guilabel:`Log in to Atlas to enable` button within the 
:guilabel:`Use natural language to generate queries and pipelines` modal. 
This modal displays after updating Compass to a version that supports 

> natural language querying and clicking the :guilabel:`Generate` 

Reviewer: This will say "Generate Query" or "Generate Aggregation" depending on the context. Is it worth spelling this out?

button on the query bar.

Before you Begin
----------------

To enable the generation of queries and aggregations in Compass, you must connect to an 

> Atlas deployment. For details on how to get a free Atlas account and 

Reviewer: this is not entirely true, it would be more accurate to say something like "you must log in with an Atlas account" 

users can use this on any type of deployment once they log in

> To enable the generation of queries and aggregations in Compass, you must connect to an 
> Atlas deployment. For details on how to get a free Atlas account and 
> deploy a free M0 cluster, see :ref:`atlas-getting-started`.

Reviewer 2: User's don't need to use an Atlas deployment with the ai feature. They only have to sign into Atlas. Once signed in it can be on any deployment.

```suggestion
To enable the generation of queries and aggregations in Compass, you must sign into 
Atlas. For details on how to get a free Atlas account and 
deploy a free M0 cluster, see :ref:`atlas-getting-started`.
```
